### PR TITLE
Truncate messages with fewer characters

### PR DESCRIPTION
**Pull request by yanksyoon**
Replace three dots characters `...` with ellipsis (U+2026) `…` characters to save 2 extra chars space.
Related issue: #3050

### CODE REVIEW

**Review by simonpasquier — APPROVED** [review]
thanks!

File: notify/util.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func Truncate(s string, n int) (string, bool) {
 	if n <= 3 {
 		return string(r[:n]), true
 	}
-	return string(r[:n-3]) + "...", true
+	return string(r[:n-1]) + "…", true
```
Comment (pstibrany) [inline notify/util.go:93]: "…" is encoded as 3 bytes in utf-8, so this doesn't save anything, and makes string longer by 2 bytes compared to previous version.

Furthermore blindly cutting last 3 or 1 byte can actually break the utf-8 sequence of bytes. Ideally string would be truncated at rune boundary.

Comment (gotjosh) [reply notify/util.go:93]: I fixed this eventually: https://github.com/prometheus/alertmanager/blob/main/notify/util.go#L88-L130

Comment (pstibrany) [reply notify/util.go:93]: Oh nice, I missed that when checking changelog of alertmanager 0.26.0.

Comment (pstibrany) [reply notify/util.go:93]: Thanks!

**Pull request merged**